src/components/screens/error-page.tsx

```diff
@@ -0,0 +1,25 @@
+import { type ReactElement } from "react";
```

Comment:
This is one options for error handling, you could also look at the old timebank or the oss-surveys-management repo I mentioned to see the ErrorHandlers which are how it is typically done

Reply:
For now we will leave it as it is